### PR TITLE
Fix/aligner

### PR DESCRIPTION
**Pull request by tadzjibov**
Fixed container endnodes and vertical aligner for tb10

### CODE REVIEW

File: src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * Bubble class that contains segments or nested bubbles.
  */
 public class Bubble implements Node, Serializable {
-    private static final long serialVersionUID = -5399272868997934829L;
+    private static final long serialVersionUID = 5065334268501873327L;
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java]: Why did you change this?

File: src/main/java/io/github/programminglife2016/pl1_2016/collapser/BubbleAligner.java [hunk 2]
```diff
@@ -42,32 +49,89 @@ private void sortByX(Collection<Node> nodes) {
...
+                    visited.add(next);
+                }
+            }
+            if (!visited.contains(main)) {
+                main.setXY(main.getX(), node.getY());
+                visited.add(main);
+            }
+        }
+
+     */
```
Comment (smesseim) [inline src/main/java/io/github/programminglife2016/pl1_2016/collapser/BubbleAligner.java]: ...

File: src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java (hunk 1 above)
Comment (tadzjibov) [inline src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java]: Otherwise it doesn't work on my pc
